src/asm/aarch64/predict.rs

```diff
@@ -110,10 +110,18 @@ pub fn dispatch_predict_intra<T: Pixel>(
           })(dst_ptr, stride, edge_ptr, w, h, ac_ptr, angle);
         }
         PredictionMode::H_PRED => {
```

Comment:
Specialising here removes the possibility of error when pasting the fallback.
```suggestion
        PredictionMode::H_PRED if angle == 180 => {
```